### PR TITLE
Remove unused bindings / namespaces

### PR DESCRIPTION
**Pull request by candid82**
Removing a few unused bindings / namespaces Joker found.

### CODE REVIEW

File: test/com/walmartlabs/test_utils.clj [hunk 1]
```diff
@@ -2,7 +2,6 @@
   (:require
     [clojure.test :refer [is]]
     [clojure.spec.test :as stest]
-    flatland.ordered.map
```
Comment (hlship) [inline test/com/walmartlabs/test_utils.clj]: This one should be left; it ensures that the OrderedMap class exists for the :import.

Comment (bcarrell) [inline test/com/walmartlabs/test_utils.clj]: I don't think that's true.

Comment (hlship) [inline test/com/walmartlabs/test_utils.clj]: The `deftype` for `OrderedMap` is in that namespace, so if you want to dependably load `OrderedMap`, the namespace should exist first. `OrderedMap` is not in the flatland jar.

Comment (candid82) [inline test/com/walmartlabs/test_utils.clj]: Ah, good catch! I put it back in.

**Pull request merged**